Client/Frontend/Browser/ZoomPageBar.swift

```diff
@@ -99,39 +125,52 @@ class ZoomPageBar: UIView {
             zoomInButton.isEnabled = false
         }
 
-        [zoomOutButton, zoomLevel, zoomInButton].forEach {
+        [zoomOutButton, separator1, zoomLevel, separator2, zoomInButton].forEach {
             stepperContainer.addArrangedSubview($0)
         }
 
         addSubviews(stepperContainer, closeButton)
     }
 
-    func setupLayout() {
+    private func setupLayout() {
+        stepperCenterXConstraint = stepperContainer.centerXAnchor.constraint(equalTo: centerXAnchor)
+        stepperLeadingConstraint = stepperContainer.leadingAnchor.constraint(equalTo: leadingAnchor,
+                                                                             constant: UX.leadingTrailingPadding)
         if isIpad {
```

Comment:
We should be checking if the device is iPad. This could cause issues in split mode or with multitasking.